docs/configuration.rst

```diff
@@ -139,34 +146,34 @@ Very few situations require you to change these settings.
 
 This feature group allows you to only lend a certain percentage of your total holding in a coin, until the lending rate suprasses a certain threshhold. Then it will lend at max capacity.
 
-``maxtolent`` is a raw number of how much you will lend of each coin whose lending rate is below ``maxtolentrate``.
+``maxtolend`` is a raw number of how much you will lend of each coin whose lending rate is below ``maxtolendrate``.
```

Comment:
I think you wanted to change this parameter name to lend, but did this only in the documentation.